src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -239,24 +220,20 @@ private ImmutableArray<ISymbol> GetSymbolsForExpressionOrStatementContext()
             // name).  If this is the case, we do not want to filter out inaccessible locals.
             var filterOutOfScopeLocals = _filterOutOfScopeLocals;
             if (filterOutOfScopeLocals)
-            {
-                if (_context.LeftToken.Parent.IsFoundUnder<LocalDeclarationStatementSyntax>(d => d.Declaration.Type))
-                {
-                    filterOutOfScopeLocals = false;
-                }
-            }
+                filterOutOfScopeLocals = !_context.LeftToken.GetRequiredParent().IsFoundUnder<LocalDeclarationStatementSyntax>(d => d.Declaration.Type);
```

Comment:
The original code is easier to understand for me. Can we just merge the if statements instead?